lib/blacklight/models/file.rb

```diff
@@ -38,6 +38,7 @@ def canvas_conversion(course)
     # Remove temporary files
     ##
     def self.cleanup
+      @@dir ||= nil
```

Comment:
As this is a class variable, you can (and probably should) initialize it outside the method, rather than here. Using `||=` with a nil looks odd (though in this case, it does work), since what you're usually after with that idiom is to assign a non-nil value if the variable is currently nil (or false).

So try something like this:

```ruby
@@dir = nil
def self.cleanup
   # etc.
end
```